Source/WebInspectorUI/UserInterface/Views/ColorSwatchDisplay.css

```diff
@@ -0,0 +1,17 @@
+.swatch-display ul{
```

Comment:
style: we prefer using `>` in order to be as specific as possible
```
.swatch-display > ul {
```

Reply:
The plan is to Inline into the Color Picker. I have another patch coming right after this, that's adding new functionality that makes it much easier to inline into Color Picker and remove the ColorSwatchDisplay all together. I kept it for this just so we could land the initial feature then improve in the second pr. This one is already changing a lot so I felt it would be hectic.